Rakefile

```diff
@@ -183,5 +189,50 @@ task :update do
   end
 end
 
+namespace :audit do
+  desc "Audit app dependencies for vulnerabilities"
+  task :dependencies do
+    branches = ["release_1.8"]
+
+    branches.each do |branch|
+      vulnerable_apps = []
+      # Audit Ruby apps
+      ruby_apps.each do |app|
+        begin
+          chdir app.path do
+            sh "git checkout #{branch}"
+            # Get last line from bundle-audit output
+            bundle_audit = %x[ bundle-audit | tail -n 1 ]
+            if bundle_audit["Vulnerabilities found!"]
+              vulnerable_apps << app.name
+            end
+            sh "git checkout -"
+          end
+        end
+      end
+
+      # Audit node.js apps
+      node_apps.each do |app|
+        begin
+          chdir app.path do
+            sh "git checkout #{branch}"
+            # Get last 3 lines from yarn audit output
+            yarn_audit = %x[yarn audit --level moderate | tail -n 3]
+            if yarn_audit["vulnerabilities"]
+              vulnerable_apps << app.name
+            end
+            sh "git checkout -"
+          end
```

Comment:
In both this and the other you place in a `begin` block. What is the thinking here?

Reply:
`sh "git checkout -"`

This probably can be removed, I added this so that when the rake task is complete my local branch is what it was before I started the rake task.